lib/nokogiri/html5/document.rb

```diff
@@ -112,9 +112,20 @@ def initialize(*args) # :nodoc:
         @url = nil
       end
 
-      # Parse a HTML5 document fragment from +tags+, returning a HTML5::DocumentFragment.
-      def fragment(tags = nil)
-        DocumentFragment.new(self, tags, root)
+      # :call-seq:
+      #   fragment() → Nokogiri::HTML5::DocumentFragment
+      #   fragment(markup) → Nokogiri::HTML5::DocumentFragment
+      #
+      # Parse a HTML5 document fragment from +markup+, returning a Nokogiri::HTML5::DocumentFragment.
+      #
+      # [Properties]
+      # - +markup+ (String) The HTML5 markup fragment to be parsed
+      #
+      # [Returns]
+      #   Nokogiri::HTML5::DocumentFragment. This object's children will be empty if `markup` is not passed, is empty, or is `nil`.
+      #
+      def fragment(markup = nil)
+        DocumentFragment.new(self, markup)
```

Comment:
Note that I changed this line to not pass in the third argument, rather than pass in an explicit `nil`.